include/libtorrent/kademlia/dht_storage.hpp

```diff
@@ -33,19 +33,17 @@ POSSIBILITY OF SUCH DAMAGE.
 #ifndef DHT_STORAGE_HPP
```

Comment:
I just noticed this. I believe all other header files have the TORRENT_ prefix for their include guards.